ui/tests/integration/components/sidebar/nav/cluster-test.js

```diff
@@ -49,12 +49,12 @@ module('Integration | Component | sidebar-nav-cluster', function (hooks) {
     });
   });
 
-  test('it should hide links and headings user does not have access too', async function (assert) {
+  test('it should hide links and headings user does not have access to', async function (assert) {
     await renderComponent();
 
     assert
       .dom('[data-test-sidebar-nav-link]')
-      .exists({ count: 2 }, 'Nav links are hidden other than secrets and dashboard');
+      .exists({ count: 3 }, 'Nav links are hidden other than secrets, recovery and dashboard');
```

Comment:
maybe add a todo since this will change, we'll want to gate these in the has-permission helper? Or we could update the [helper api path map](https://github.com/hashicorp/vault/blob/main/ui/app/services/permissions.js#L15) with the relevant routes and wrap the sidenav link now - up to you :)